Dockerfile

```
@@ -52,6 +52,10 @@ ENV SUPERVISOR_VERSION=4.2.2
 ENV POETRY_VERSION=1.5.1
 ENV POETRY_URL="https://install.python-poetry.org"
 ENV POETRY_HOME="/etc/poetry"
+# required for postgres ssl: the crt file doesn't exist
+# but the path must point to a visible directory otherwise we
+# get a permissions error
+ENV PGSSLCERT /tmp/postgresql.crt
```

Should use the new environment variable format to resolve the deprecation warning.
```suggestion
ENV PGSSLCERT=/tmp/postgresql.crt
```